docs/localization.md

```diff
@@ -160,4 +160,4 @@ let localizedCount = NumberFormatter.localizedString(from: NSNumber(value: count
 
 ## Testing considerations
 
-Test changes that include localized content by using large words or with letters/accents not frequently used in English.
\ No newline at end of file
+Test changes that include localized content by using large words or with letters/accents not frequently used in English.
```

Comment:
This looks like a whitespace changed automatically created by the GitHub in-browser editor.